### PR TITLE
Vrrm

### PR DESCRIPTION
**Pull request by mike-f50**

### CODE REVIEW

File: HelpMyStreetFE/HelpMyStreetFE/Models/Account/Jobs/JobDetail.cs [hunk 1]
```diff
@@ -6,6 +6,7 @@ namespace HelpMyStreetFE.Models.Account.Jobs
 {
     public class JobDetail
     {
+        public RequestSummary RequestSummary { get; set; }
```
Comment (jawwadmukhtargmail) [inline HelpMyStreetFE/HelpMyStreetFE/Models/Account/Jobs/JobDetail.cs:9]: Does this model class need to be defined here when it's already defined in the nuget package

File: HelpMyStreetFE/HelpMyStreetFE/Models/Account/Jobs/RequestDetail.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using HelpMyStreet.Utils.Models;
+
+namespace HelpMyStreetFE.Models.Account.Jobs
+{
+    public class RequestDetail
```
Comment (jawwadmukhtargmail) [inline HelpMyStreetFE/HelpMyStreetFE/Models/Account/Jobs/RequestDetail.cs:9]: Same again. Does this need to defined here

File: HelpMyStreetFE/HelpMyStreetFE/Models/Account/Jobs/JobDetail.cs (hunk 1 above)
Comment (mike-f50) [reply HelpMyStreetFE/HelpMyStreetFE/Models/Account/Jobs/JobDetail.cs:9]: I think RequestSummary here is the one from NuGet.

File: HelpMyStreetFE/HelpMyStreetFE/Models/Account/Jobs/RequestDetail.cs (hunk 2 above)
Comment (mike-f50) [reply HelpMyStreetFE/HelpMyStreetFE/Models/Account/Jobs/RequestDetail.cs:9]: Good shout.  That class is identical to GetRequestDetailsResponse so I'll remove mine.

**Review by jawwadmukhtargmail — APPROVED** [review]
approved - just scanned over rather than looking at every line

**Pull request merged**